### PR TITLE
Retire change wave 17.4

### PR DESCRIPTION
**Pull request by rainersigwald**
Its time has come, per the documented schedule.

### CODE REVIEW

**Comment by rainersigwald** [comment]
Question for @baronfel and @marcpopMSFT: does the newer "try to stack breaking changes on .NET Core releases" policy imply that we should instead require all stale changewaves in the `X.0.100` release, instead of an even MSBuild release as we have been doing?

**Comment by marcpopMSFT** [comment]
You mean basically retire the change waves once a year in November instead of every 6 months? I guess retiring a change wave is breaking so maybe that makes sense? Would we retire two waves each year?

**Review by rokonec — APPROVED** [review]

File: src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs [hunk 1]
```diff
@@ -450,21 +389,6 @@ private object GetResolverState(int submissionId, SdkResolver resolver)
...
-            lock (_lockObject)
-            {
-                if (_resolversList != null)
-                {
-                    return;
-                }
-
-                MSBuildEventSource.Log.SdkResolverServiceInitializeStart();
-                _resolversList = _sdkResolverLoader.LoadAllResolvers(location);
-                MSBuildEventSource.Log.SdkResolverServiceInitializeStop(_resolversList.Count);
```
Comment (ladipro) [inline src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs:464]: nit: `LoadAllResolvers` is now unused in product code and so are the two event source calls. It looks like `LoadResolversFromManifest` and `LoadResolvers` become transitively dead as well.

**Review by ladipro — APPROVED** [review]

**Review by Forgind — APPROVED** [review]

**Comment by rainersigwald** [comment]
Closing to wait for 9.0

**Pull request closed without merging**